src/Forms/Shared/Samples/Geometry/ListTransformations/ListTransformations.xaml.cs

```diff
@@ -134,7 +134,7 @@ private void GetSuitableTransformations(SpatialReference inSpatialRef, SpatialRe
             {
                 DatumTransformationListBoxItem item = new DatumTransformationListBoxItem(transform)
                 {
-                    IsDefault = (transform.Name == defaultTransform.Name)
+                    IsDefault = transform.Name == defaultTransform.Name
```

Comment:
Keep parens in cases where there is both assignment and comparison in the same statement. It's much easier to read at a glance, IMO.